### ISSUE TITLE
Failed to start proxy

### ISSUE THREAD

**Issue by basnijholt** [jupyter/jupyterhub#237]
I have Jupyterhub installed on two computers, both give me the same error:

```
bnijholt@qclab-hpc1:~$ sudo jupyterhub
[I 2015-04-23 15:12:13.581 JupyterHub app:513] Loading cookie_secret from /home/bnijholt/jupyterhub_cookie_secret
[W 2015-04-23 15:12:13.608 JupyterHub app:251] 
    Generating CONFIGPROXY_AUTH_TOKEN. Restarting the Hub will require restarting the proxy.
    Set CONFIGPROXY_AUTH_TOKEN env or JupyterHub.proxy_auth_token config to avoid this message.
    
[I 2015-04-23 15:12:13.614 JupyterHub app:614] Not using whitelist. Any authenticated user will be allowed.
[I 2015-04-23 15:12:13.622 JupyterHub app:993] Hub API listening on http://localhost:8081/hub/
[I 2015-04-23 15:12:13.626 JupyterHub app:749] Starting proxy @ http://*:8000/

util.js:555
  ctor.prototype = Object.create(superCtor.prototype, {
                                          ^
TypeError: Cannot read property 'prototype' of undefined
    at Object.exports.inherits (util.js:555:43)
    at Object.<anonymous> (/usr/local/lib/node_modules/configurable-http-proxy/node_modules/http-proxy/lib/http-proxy/index.js:111:17)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/configurable-http-proxy/node_modules/http-proxy/lib/http-proxy.js:4:17)
    at Module._compile (module.js:456:26)
[C 2015-04-23 15:12:14.636 JupyterHub app:999] Failed to start proxy
    Traceback (most recent call last):
      File "/usr/local/lib/python3.4/dist-packages/jupyterhub/app.py", line 997, in start
        yield self.start_proxy()
      File "/usr/local/lib/python3.4/dist-packages/jupyterhub/app.py", line 770, in start_proxy
        _check()
      File "/usr/local/lib/python3.4/dist-packages/jupyterhub/app.py", line 766, in _check
        raise e
    RuntimeError: Proxy failed to start with exit code 8
```

This is what I get by just running configurable-http-proxy
```
bnijholt@qclab-hpc1:~/jupyterhub$ configurable-http-proxy 

util.js:555
  ctor.prototype = Object.create(superCtor.prototype, {
                                          ^
TypeError: Cannot read property 'prototype' of undefined
    at Object.exports.inherits (util.js:555:43)
    at Object.<anonymous> (/usr/local/lib/node_modules/configurable-http-proxy/node_modules/http-proxy/lib/http-proxy/index.js:111:17)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/configurable-http-proxy/node_modules/http-proxy/lib/http-proxy.js:4:17)
    at Module._compile (module.js:456:26)
```

**Comment by basnijholt** [comment]
The suggestions given here: http://lists.ipython.scipy.org/pipermail/ipython-dev/2014-October/015257.html don't do the trick. 

I've reinstalled everything that needs to be installed in order for Jupyterhub to work.

**Comment by basnijholt** [comment]
https://github.com/jupyter/configurable-http-proxy/issues/25

is the same problem I think

**Comment by basnijholt** [comment]
I fixed it by 
```sudo npm uninstall -g configurable-http-proxy ```
```git clone https://github.com/jupyter/configurable-http-proxy.git```

adding this in package.json
```"dependencies": {
    "eventemitter3": "0.1.6",```

Such that it looks like:

```
{
  "version": "0.2.2-dev",
  "name": "configurable-http-proxy",
  "description": "A configurable-on-the-fly HTTP Proxy",
  "author": "Jupyter Developers",
  "license": "BSD",
  "repository": {
    "type": "git",
    "url": "https://github.com/jupyter/configurable-http-proxy.git"
  },
  "dependencies": {
    "commander": "~2",
    "eventemitter3": "0.1.6",
    "http-proxy": "~1.6",
    "winston": "~0.8",
    "strftime": "~0.8"
  },
  "devDependencies": {
    "jasmine-node": "~2",
    "request": "~2",
    "ws": "~0.4"
  },
  "main": "index.js",
  "files": [
    "index.js",
    "lib/configproxy.js",
    "lib/trie.js",
    "bin/configurable-http-proxy"
  ],
  "bin": {
    "configurable-http-proxy": "bin/configurable-http-proxy"
  },
  "scripts": {
    "test": "jasmine-node --captureExceptions test"
  }
}
```
Then 
```cd configurable-http-proxy```
```sudo npm install -g .```

**Comment by minrk** [comment]
This is fixed in configurable-http-proxy 0.2.2
